plugins/modules/discovery_intent.py

```diff
@@ -981,7 +1011,7 @@ def main():
         dnac_discovery.msg = "State {0} is invalid".format(state)
         dnac_discovery.check_return_status()
 
-    dnac_discovery.validate_input().check_return_status()
+    dnac_discovery.validate_input(state=state).check_return_status()
```

Comment:
Any reasons for this
 state=state ?

Reply:
We are passing the state value here to check in the validation part

Reply:
but why do we need to assign state = state when we call the function?

Reply:
Will it work if I don't pass the state?